### ISSUE TITLE
cp: option not recognized: p

### ISSUE THREAD

**Issue by michael-ts** [shelljs/shelljs#1086]
### Node version (or tell us if you're using electron or some other framework):
12.21.0

### ShellJS version (the most recent version/Github branch you see the bug on):
0.8.5

### Operating system:
Linux

### Description of the bug:
Copying fails when trying to use the `-p` option.

### Example ShellJS command to reproduce the error:

```
var shell = require("shelljs")
shell.cp("-p",path_to_file,dest_dir)
```
Output when running from node shell:
```
cp: option not recognized: p
[String: ''] {
  stdout: '',
  stderr: 'cp: option not recognized: p',
  code: 1,
  cat: [Function: bound ],
  exec: [Function: bound ],
  grep: [Function: bound ],
  head: [Function: bound ],
  sed: [Function: bound ],
  sort: [Function: bound ],
  tail: [Function: bound ],
  to: [Function: bound ],
  toEnd: [Function: bound ],
  uniq: [Function: bound ]
}
```

**Comment by nfischer** [comment]
The `-p` flag was only landed recently: https://github.com/shelljs/shelljs/pull/869. This will be available in v0.9.0.

**Comment by cdwmhcc** [comment]
> The `-p` flag was only landed recently: #869. This will be available in v0.9.0.

where

**Comment by nfischer** [comment]
Can you try this out in the latest ShellJS release (v0.9.1) and let me know if that fixed the problem?
